File: CS/WpfApplication/MyGridControl/MyCustomView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Xpf.Grid;
using System.Windows;

namespace WpfApplication.MyGridControl
{
    public class MyCustomView : TableView
    {
        protected override void OnMouseMove(System.Windows.Input.MouseEventArgs e)
        {
            base.OnMouseMove(e);
            CheckHotTrack(e);
        }

        private void CheckHotTrack(System.Windows.Input.MouseEventArgs e)
        {
            TableViewHitInfo hi =    CalcHitInfo(e.OriginalSource as DependencyObject);
            if (hi.RowHandle != GridControl.InvalidRowHandle)
                FocusedRowHandle = hi.RowHandle;
        }

        
        
    }
}

File: VB/WpfApplication/MyGridControl/MyCustomView.vb
Imports DevExpress.Xpf.Grid
Imports System.Windows

Namespace WpfApplication.MyGridControl

    Public Class MyCustomView
        Inherits TableView

        Protected Overrides Sub OnMouseMove(ByVal e As Input.MouseEventArgs)
            MyBase.OnMouseMove(e)
            CheckHotTrack(e)
        End Sub

        Private Sub CheckHotTrack(ByVal e As Input.MouseEventArgs)
            Dim hi As TableViewHitInfo = CalcHitInfo(TryCast(e.OriginalSource, DependencyObject))
            If hi.RowHandle <> DataControlBase.InvalidRowHandle Then FocusedRowHandle = hi.RowHandle
        End Sub
    End Class
End Namespace
